lang/operators.talon

```diff
@@ -42,9 +42,7 @@ op mod equals: user.code_operator_modulo_assignment()
 
 #bitwise operators
 [op] bitwise and: user.code_operator_bitwise_and()
-(op | logical | bitwise) and equals: user.code_operator_bitwise_and_equals()
```

Comment:
Generally, our approach is to leave these commands here, even if not all languages support them. Not perfect.

Reply:
This was dead code, "code_operator_bitwise_and_equals" Was never implemented anywhere.